specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -361,6 +381,58 @@ function MergeButton<SCHEMA extends AnySchema>({
   );
 }
 
+export function Status({
```

Comment:
(optional) can you take a look WbStatus component and see if any code can be reused/encapsulated or if it's too diferent